messenger/messenger.go

```diff
@@ -316,30 +320,14 @@ func sendMessage(msg string) {
 		}
 		return
 	}
+	msg = fmt.Sprintf("`[%s]` %s", hostname, msg)
 	sendRawMessage(encodeMessage(msg))
 }
 
 func sendRawMessage(msg []byte) {
 	if webhookURL == "" || msgCh == nil {
 		return
 	}
-	/*
-		if len(msg) > maxMessageLen {
-			for {
-				l := maxMessageLen - 10
-				if l > len(msg) {
-					l = len(msg)
-				}
-				msg1 := msg[:l]
-				msg = msg[l:]
-				sendMessage(msg1)
-				if len(msg) == 0 {
-					break
-				}
-			}
-			return
-		}
-	*/
```

Comment:
Why commented?

Reply:
It was commented before and exactly the same code that is present in the other `sendMessage` function (this is `sendMessageRaw`). I assumed it was just moved there at some point but the commented code stayed here, so I removed it.